packages/contracts/src/abis/mainnet/StakingEscrow.js

```diff
@@ -1,2014 +1,765 @@
-          "type":"function"
-       },
-       {
-          "inputs":[
-             {
-                "internalType":"uint256",
-                "name":"_blockNumber",
-                "type":"uint256"
-             }
-          ],
-          "name":"totalStakedAt",
-          "outputs":[
-             {
-                "internalType":"uint256",
-                "name":"",
-                "type":"uint256"
-             }
-          ],
-          "stateMutability":"view",
-          "type":"function"
-       },
-       {
-          "inputs":[
-             {
-                "internalType":"address",
-                "name":"_owner",
-                "type":"address"
-             },
-             {
-                "internalType":"uint256",
-                "name":"_blockNumber",
-                "type":"uint256"
-             }
-          ],
-          "name":"totalStakedForAt",
-          "outputs":[
-             {
-                "internalType":"uint256",
-                "name":"",
-                "type":"uint256"
-             }
-          ],
-          "stateMutability":"view",
-          "type":"function"
-       },
-       {
-          "inputs":[
-
-          ],
-          "name":"totalSupply",
-          "outputs":[
-             {
-                "internalType":"uint128",
-                "name":"",
-                "type":"uint128"
-             }
-          ],
-          "stateMutability":"view",
-          "type":"function"
-       },
-       {
-          "inputs":[
-             {
-                "internalType":"address",
-                "name":"newOwner",
-                "type":"address"
-             }
-          ],
-          "name":"transferOwnership",
-          "outputs":[
-
-          ],
-          "stateMutability":"nonpayable",
-          "type":"function"
-       },
-       {
-          "inputs":[
-             {
-                "internalType":"address",
-                "name":"_testTarget",
-                "type":"address"
-             }
-          ],
-          "name":"verifyState",
-          "outputs":[
-
-          ],
-          "stateMutability":"nonpayable",
-          "type":"function"
-       },
-       {
-          "inputs":[
-             {
-                "internalType":"uint256",
-                "name":"_value",
-                "type":"uint256"
-             }
-          ],
-          "name":"withdraw",
-          "outputs":[
-
-          ],
-          "stateMutability":"nonpayable",
-          "type":"function"
-       },
-       {
-          "inputs":[
-
-          ],
-          "name":"workLock",
-          "outputs":[
-             {
-                "internalType":"contract WorkLockInterface",
-                "name":"",
-                "type":"address"
-             }
-          ],
-          "stateMutability":"view",
-          "type":"function"
-       }
-    ]
- ];
\ No newline at end of file
+   "StakingEscrow",
+   "v5.7.1",
+   "0x18eFb520dA5D387982C860a64855C14C0AcADF3F",
```

Comment:
This isn't the latest address or ABI on mainnet.

^ @cygnusv

Reply:
I will update it.  It's not actually used or displayed anywhere but probably good to keep these things current.